src/main/java/ToDo.java

```diff
@@ -0,0 +1,17 @@
+public class ToDo extends Task {
+    public ToDo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.getStatusIcon() + " " + super.getDescription();
```

Comment:
I like the use of the `getStatusIcon()` and `getDescription()` methods for overriding the child class `toString()` method, however you could consider implementing the `toString()` method for the `Task` class as its a more conventional way to do it and it gets the job done just as well!